Sources/Private/Experimental/Layers/ShapeItemLayer.swift

```diff
@@ -111,6 +141,28 @@ extension ShapeItemLayer: AnimationLayer {
 
     // TODO: Support `lineDashPhase` and `lineDashPattern`
   }
+
+  private func addAnimations(for trim: Trim, context: LayerAnimationContext) {
+    addAnimation(
+      for: .strokeStart,
+      keyframes: trim.start.keyframes,
+      value: { strokeStart in
+        // Lottie animation files express stoke trims as a numerical percentage value
+        // (e.g. 25%, 50%, 100%) so we divide by 100 to get the decimal values
+        // expected by Core Animation (e.g. 0.25, 0.5, 1.0).
+        CGFloat(strokeStart.cgFloatValue) / 100
```

Comment:
Interesting idea -- one thought here is that the code itself is already very lightweight imo (just a `/ 100`), which makes me think that we may not get much out of having a shared utility for this.

Reply:
Yeah, I'm mostly thinking about the "idea" of combining percentage normalization so that this comment/logic doesn't need to be sprinkled around everywhere. Just an idea of course, no obligation to take this approach.